commands/aliases.go

```diff
@@ -20,9 +20,24 @@ import (
 var (
 	// note that this is relative to the $(pwd) as opposed to the `commands` directory
 	aliasStorageFilepath = "aliases.json"
-	aliasMap             = make(map[string]string)
+	aliasMap             = make(map[string]*alias)
 )
 
+type aliasKind int
+
+const (
+	// if adding a new kind, add it *after* the last one in the list or you will
+	// break the existing `aliases.json` file
+	kindIMAGE aliasKind = iota
+	kindOTHER
+)
+
+// alias represents the alias value and meta information used to display the alias
+type alias struct {
+	Value string    `json: "Value"`
+	Kind  aliasKind `json: "Kind"`
```

Comment:
Two points:
- PascalCase in JSON? ;P hmm
- The tags arent needed if the text is the same as the field names. Could argue that it makes no difference if they *are* there, just thought id bring it up

Reply:
I will change it to lower camel if this pleases you :joy: 

I will still need the tags though because the `aliasMap` is encoded as json as well and the default when encoding is to name them with an uppercase letter at the start.

See https://blog.golang.org/json-and-go

Reply:
be grand either way haha